### ISSUE TITLE
Ambridge-2019-60: norm data on grammaticality judgments on different languages

### ISSUE THREAD

**Issue by LinguList** [concepticon/concepticon-data#677]
I guess this is the first time somebody who's not any of us uses concepticon for their research!

https://osf.io/pavm7/

They write:

> We selected from the Action/Process category of http://concepticon.clld.org/
> (List, Cysouw & Forkel, 2016) 60 items that (a) are lexicalized as common verbs in each of
> the five languages, (b) are familiar to young children, (c) span a range of direct (external) and
> indirect (internal) causation, and (d) can be easily depicted in animations, such that naïve
> participants are able to guess the intended verb (or a close synonym). This was achieved
> through successive rounds of piloting, using incorrect guesses to refine the animations and
> discard unsuitable candidate verbs. Details of the verb-guessing pilot can be found in
> Appendix 1.

Paper is here: https://osf.io/mqgw9/download

**Comment by Kristina-Pianykh** [comment]
@LinguList should this list be added to Concepticon?

**Comment by LinguList** [comment]
Yes! I think that would be very nice!

**Comment by LinguList** [comment]
Ah, @Kristina-Pianykh, if you add this, could you make sure to also add as much of their specific data, as possible? That's ultimately what would be interesting for us.

**Comment by Kristina-Pianykh** [comment]
The data contain two separate columns for causal and non-causal interpretations for each of the 60 verbs. E.g., _bark_ in the non-causal column corresponds to _make bark_ in the causal column and so on. For some instances, there are separate concepts in Concepticon (e.g., BOIL (OF LIQUID) and BOIL (SOMETHING)) but they are rather scarce for such a small list (compare also with _laugh_ and _make laugh_)

If it's not an issue, then there are basically two options: either leave the two columns as they are and map each entry to a separate concept (which would result in two respective concept set columns). Or put all the entries together into one column and map correspondingly. @LinguList what do you think?

**Comment by LinguList** [comment]
Okay, here's what I propose: if possible, we should separate causatives from non-causative constructions, since this is again valuable information. If we lack the forms in concepticon, I'd propose to add them (even if it's tedious). Furthermore, you can add a column "counterpart", which will store the counterpart concept (e.g., a causative form of a non-causative form), and this could be a concepticon ID or the concepticon_gloss. Does that sound okay?

**Comment by Kristina-Pianykh** [comment]
Yes, but in this case adding all the missing concepts would mean also that there will be such awkward concepts like MAKE LAUGH, MAKE CLIMB, MAKE CRY, MAKE DANCE, etc. for which there are no full-fledged lexical realizations in English. Would it still be fine?

**Comment by LinguList** [comment]
Yes, I think it would be very fine, since there are some other unmapped datasets where causatives will occur. And since this is a first list where people actually *query* that information, it is also "licensed", as our rule is: we can add a certain concept, if there's a questionnaire that makes the distinction. Here, we could say that this is the case.

BTW: maybe, the information on SOURCE is not even needed, but what you rather need is to add the causative information in the `conceptrelations.tsv`. It would be good to communicate this with @Schweikhard, who wanted to look into enhancing the concept relations anyway (and the bug is fixed now which prevented us from adding more than just one conceptrelation for the same concept pair).

**Comment by AnnikaTjuka** [comment]
https://github.com/concepticon/concepticon-data/issues/925 mentions the underlying article for this issue. I changed the header accordingly and closed issue #925.

**Comment by mathildavz** [comment]
@LinguList I've started working on this issue now. I can see that there has been quite some discussion about this list already. As a first question, I struggle to find the correct verb list. Via Appendix 1 in the paper, I've reached the same page you linked above. However, on there, I found several excel files in the "JudgmentLists" folder that contain verb lists, none of which actually has 60 verbs or relevant specific data.

**Comment by LinguList** [comment]
It is always tedious to look at OSF repositories.
